static/css/profile.css

```diff
@@ -6,11 +6,10 @@ main {
   margin-top: 6rem;
 }
 
-main > header {
+main header {
```

Comment:
We only want this applied on the header that is a direct child of main.
We do have a `header` in `article` in the list of articles; we don't want these styles there.